src/MTGCore.ClientApp/ClientApp/src/components/SearchFilterModal.vue

```diff
@@ -0,0 +1,130 @@
+                <input v-model="form.Colours" class="form-check-input" type="checkbox" value="W" id="chkWhite" />
+                <label class="form-check-label" for="chkWhite">W</label>
+              </div>
+              <div class="form-check form-inline col-md-2">
+                <input v-model="form.Colours" class="form-check-input" type="checkbox" value="U" id="chkBlue" />
+                <label class="form-check-label" for="chkBlue">U</label>
+              </div>
+              <div class="form-check form-inline col-md-2">
+                <input v-model="form.Colours" class="form-check-input" type="checkbox" value="B" id="chkBlack" />
+                <label class="form-check-label" for="chkBlack">B</label>
+              </div>
+              <div class="form-check form-inline col-md-2">
+                <input v-model="form.Colours" class="form-check-input" type="checkbox" value="R" id="chkRed" />
+                <label class="form-check-label" for="chkRed">R</label>
+              </div>
+              <div class="form-check form-inline col-md-2">
+                <input v-model="form.Colours" class="form-check-input" type="checkbox" value="G" id="chkGreen" />
+                <label class="form-check-label" for="chkGreen">G</label>
+              </div>
+              <div class="form-check form-inline col-md-2">
+                <input v-model="form.Colours" class="form-check-input" type="checkbox" value="C" id="chkColourless" />
+                <label class="form-check-label" for="chkColourless">C</label>
+              </div>
+            </div>
+
+            <hr />
+
+            <div class="row form-group">
+              <div class="col-lg-2">Rarity:</div>
+              <div class="col-lg-10">
+                <select v-model="form.Rarity" class="form-control form-select form-select-lg">
+                  <option disabled selected value> -- select an option -- </option>
+                  <option value="Uncommon">Uncommon</option>
+                  <option value="Common">Common</option>
+                  <option value="Mythic Rare">Mythic Rare</option>
+                  <option value="Rare">Rare</option>
+                </select>
+              </div>
+            </div>
+
+            <div class="row form-group">
+              <div class="col-lg-2">Set:</div>
+              <div class="col-lg-10">
+                <input v-model="form.Set" type="text" class="form-control" />
+              </div>
+            </div>
+
+            <div class="row form-group">
+              <div class="col-lg-2">Price:</div>
+              <div class="col-lg-10">
+                <input v-model="form.Price" type="text" class="form-control" />
+              </div>
+            </div>
+          </div>
+          <div class="modal-footer">
+            <button type="button" class="btn btn-warning">Clear</button>
+            <button v-on:click="submitFilter" type="button" class="btn btn-primary">Submit</button>
+          </div>
```

Comment:
The button is labelled 'submit' which I think is confusing. After clicking this button, I think it should initiate the process of retrieving cards.